### ISSUE TITLE
sigmoid is missing in QuestionAttnGRU.py

### ISSUE THREAD

**Issue by keithhans** [YerevaNN/R-NET-in-Keras#14]
According to equation 6 in the paper, there should be a sigmoid on K.dot(GRU_inputs, W_g1).

**Comment by mahnerak** [comment]
Completely agree with you. Will be fixed soon.

**Comment by keithhans** [comment]
Just noticed it is also missing in SelfAttnGRU.py

**Comment by mahnerak** [comment]
We also noticed that.
Sigmoid disappeared after we switched from using Dense layers (with sigmoid activation) to SharedWeight layers.
Now I'm trying to repeat training process. We hope can get better scores after fixing this issue.

**Comment by mahnerak** [comment]
Pull request is merged.
Not closing the issue until we compare new scores.
@MartinXPN please rerun the instruction steps from README.md
